### PR TITLE
AoC GCP Add a fix users permissions step for Linux users

### PR DESCRIPTION
**Pull request by thenets**
Missing step whenever we run the command with the `--output-data-file` parameter.

### CODE REVIEW

File: stories/topics/proc-gcp-create-data-file.adoc [hunk 1]
```diff
@@ -9,9 +9,21 @@
 ----
 docker run --rm -v $(pwd)/command_generator_data/:/data $IMAGE command_generator_vars gcp_backup_deployment --output-data-file /data/backup.yml
 ----
+
+. (Linux users) Fix user's permissions:
```
Comment (ianf77) [inline stories/topics/proc-gcp-create-data-file.adoc:13]: Fix permissions for Linux users

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc [hunk 2]
```diff
@@ -18,6 +18,18 @@ The use of `gcp_backups_delete` is described in this section.
 ----
 docker run --rm -v $(pwd)/command_generator_data/:/data $IMAGE command_generator_vars gcp_backups_delete --output-data-file /data/backups_delete.yml
 ----
+
+. (Linux users) Fix user's permissions:
```
Comment (ianf77) [inline stories/topics/proc-gcp-deleting-backups-playbook.adoc:22]: Fix permissions for Linux users

File: stories/topics/proc-gcp-generate-variables.adoc [hunk 3]
```diff
@@ -20,31 +20,26 @@ $ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE \
 command_generator_vars gcp_setup_logging_monitoring \
 --output-data-file /data/logging-monitoring.yml
 ----
-+
-Creates the following command file:
-+
+
+. (Linux users) Fix user's permissions:
```
Comment (ianf77) [inline stories/topics/proc-gcp-generate-variables.adoc:24]: Fix permissions for Linux users

File: stories/topics/proc-gcp-listing-backups-playbook.adoc [hunk 4]
```diff
@@ -10,23 +10,20 @@ This playbook enables you to list the existing backups in a specific bucket.
 ----
 docker run --rm -v $(pwd)/command_generator_data/:/data $IMAGE command_generator_vars gcp_backup_list --output-data-file /data/backups_list.yml
 ----
-+
-Produces the following output:
-+
+
+. (Linux users) Fix user's permissions:
```
Comment (ianf77) [inline stories/topics/proc-gcp-listing-backups-playbook.adoc:14]: Fix permissions for Linux users

File: stories/topics/proc-gcp-create-data-file.adoc [hunk 5]
```diff
@@ -9,9 +9,21 @@
...
+[literal, options="nowrap" subs="+attributes"]
++
+----
+# Change the owner of the command_generator_data directory
+sudo chown -R $USER:$USER command_generator_data/
```
Comment (ianf77) [inline stories/topics/proc-gcp-create-data-file.adoc:18]: $ chown -R $USER....

File: stories/topics/proc-gcp-listing-backups-playbook.adoc [hunk 6]
```diff
@@ -10,23 +10,20 @@ This playbook enables you to list the existing backups in a specific bucket.
...
 ----
-Playbook: gcp_backup_list
-Description: This playbook list the backups in a given bucket.
------------------------------------------------
-This playbook list the backups in a given bucket.
-
------------------------------------------------
-Command generator template: 
+# Change the owner of the command_generator_data directory
+sudo chown -R $USER:$USER command_generator_data/
```
Comment (ianf77) [inline stories/topics/proc-gcp-listing-backups-playbook.adoc:19]: # chown -R ...

**Review by ianf77 — COMMENTED** [review]
a few comments

File: stories/topics/proc-gcp-create-data-file.adoc [hunk 7]
```diff
@@ -9,9 +9,21 @@
 ----
 docker run --rm -v $(pwd)/command_generator_data/:/data $IMAGE command_generator_vars gcp_backup_deployment --output-data-file /data/backup.yml
 ----
+
+. (Linux users) Fix user's permissions:
+[literal, options="nowrap" subs="+attributes"]
++
```
[the remark this answers is not in the document]
Comment (thenets) [reply stories/topics/proc-gcp-create-data-file.adoc]: I cleaned all the empty spaces

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**